stock_voucher/models/stock_picking.py

```diff
@@ -17,27 +17,25 @@ class StockPicking(models.Model):
         copy=False,
```

Comment:
remove  from math import ceil, this one is not used in the file anymore and generate pylint error  F401 'imported but unused